components/tools/OmeroPy/src/omero/tables.py

```diff
@@ -217,9 +217,9 @@ def __init__(self, file_path, hdf5lock):
     def size(self):
         return self.__hdf_path.size
 
-    def openfile(self, mode):
+    def openfile(self, mode, policy='default'):
+        tables.file._FILE_OPEN_POLICY = policy
```

Comment:
`tables.file._FILE_OPEN_POLICY` is a global module level variable. Do you know in what situations it isn't already set to `default`?